content/blog/how-to-analyze-grpc-with-wireshark.md

```diff
@@ -0,0 +1,157 @@
+---
+title: How to analyze gRPC with Wireshark
+date: 2020-10-20
+spelling: cSpell:ignore Huang Qiangxiong
+author:
+  name: Huang Qiangxiong
+  link: https://github.com/huangqiangxiong
+---
+
+[Wireshark](https://www.wireshark.org/) is the most famous open source network protocol analyzer, which is widely used in network troubleshooting, analysis, software and communications protocol development, and education. With Wireshark, people can analyze the messages of gRPC protocol that transferred over the wire (network), or learning the binary wire format of these protocols. This tutorial will show you how to configure and use Wireshark gRPC dissector to analyze gRPC with *.proto files. Note that the gRPC dissector actually relies on the Protocol Buffers (Protobuf) dissector, and some functions (including setting *.proto files paths) are accomplished through the Protobuf dissector. Wireshark started to support parsing gRPC (and Protobuf) in version 2.6.0, and some new features (including supporting *.proto* files) have been added in later versions. You can refer to section [History of Wireshark gRPC and Protobuf Dissectors](#History of Wireshark gRPC and Protobuf Dissectors) for details. Wireshark 3.4.0 is the latest stable version, is also the version we used to introduce examples later.
```

Comment:
If you look at the preview, you'll see that this paragraph doesn't render correctly, in particular, because of this link:

    [History of Wireshark gRPC and Protobuf Dissectors](#History of Wireshark gRPC and Protobuf Dissectors)

This is the proper anchor name: `#history-of-wireshark-grpc-and-protobuf-dissectors`

Reply:
done